### DISCUSSION TITLE
Migrating to Python3

### DISCUSSION THREAD

**Discussion by MazdaX** [GenomeRIK/tama#50] [General]
Hi Rick,
Thanks for all the amazing Wiki and detailed code as always. Just a question about your migration plans to python3. Do you have a timeline for this? 

I'm currently in the process of wrapping your tools and couple of PacBio ones (CCS, Lima and isoseq3 refine) in to a NextFlow / Docker container. I was wondering if you have this converting to python 3 in the works or not. 

Thanks

**Comment by GenomeRIK** [comment]
Hi Mazdak!

I do have plans of migrating to Python 3 but I do not have a timeline for this right now. If you need this done by a certain time can you let me know when that is?

Thank you,
Richard

**Reply by MazdaX** [reply]
Hi Rick,

It's obviously not a pressing timeline but it will hopefully happen before the end of 2021. It will be mainly a wrapping workflow for academic use and Ofcourse one can always release a updated version. 

Cheers,
Mazdak

**Reply by GenomeRIK** [reply]
Ah yes, definitely plan on getting to Python 3 before the end of this year! Also lot's of updates in the work and new tools!

**Reply by MazdaX** [reply]
Amazing. Looking forward to it.
